model/src/test/kotlin/LicenseFindingTest.kt

```diff
@@ -119,5 +119,29 @@ class LicenseFindingTest : WordSpec({
             "AGPL-1.0-or-later OR BSD (3-Clause) OR BSD (2-Clause)".mapLicense(mapping) shouldBe
                     "AGPL-1.0-or-later OR BSD-3-Clause OR BSD-2-Clause"
         }
+
```

Comment:
commit-message: Please indicate whether this commit is non-functional change, e.g. does it just remove code redundancy?